stl/inc/algorithm

```diff
@@ -72,6 +72,160 @@ struct _Optimistic_temporary_buffer { // temporary storage with _alloca-like att
     aligned_union_t<0, _Ty> _Stack_space[_Optimistic_count];
 };
 
+#ifdef __cpp_lib_concepts
+namespace ranges {
+    // CONCEPT _Convertible_from
+    template <class _To, class _From>
+    concept _Convertible_from = convertible_to<_From, _To>;
+
+#ifdef __clang__
+#pragma clang diagnostic push
+#pragma clang diagnostic ignored "-Wunknown-attributes"
+#endif // __clang__
+
+    // STRUCT TEMPLATE in_fun_result
+    template <class _In, class _Fn>
+    struct in_fun_result {
+        [[no_unique_address]] _In in;

```

Comment:
Did c1xx ever land on implementing this in a way we could actually use?

Reply:
C1XX ignores, and clang (as you may have guessed) pretends not to recognize the attribute presumably because they don't yet know what ABI behavior to emulate. There's really no reason to use the attribute at all except that (1) that's what the spec says, and (2) I WANT MY NO_UNIQUE_ADDRESS AND I'LL USE IT EVEN IF IT DOES NOTHING AT ALL.